taxi/forms.py

```diff
@@ -0,0 +1,60 @@
+from django.contrib.auth import get_user_model
+from django.contrib.auth.forms import UserCreationForm
+from django import forms
+from django.core.exceptions import ValidationError
+
+from taxi.models import Driver, Car
+
+
+class DriverLicenseUpdateForm(forms.ModelForm):
+    MIN_LENGTH = 8
```

Comment:
I believe it’s better to call this variable something like required length or expected length as now it is could be misleading that license number can be longer than 8.